pickhardtpayments/OracleLightningNetwork.py

```diff
@@ -78,21 +102,25 @@ def theoretical_maximum_payable_amount(self, source: str, destination: str, base
         mincut, _ = nx.minimum_cut(test_network, source, destination)
         return mincut
 
-    def settle_payment(self, path: List[OracleChannel], payment_amount: int):
+    def settle_payment(self, attempt: Attempt):
```

Comment:
I agree with the name change.
in Payment.settle(), you call this when looping through the INFLIGHT attempts only. What I like about this way is that the code is more instructive. it can be easier understood how the settlement in the payment works.